lib/Service/Group/ConnectedGroupsService.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+/**
+ * @copyright Copyright (c) 2024 Arawa
+ *
+ * @author 2024 Sébastien Marinier <seb@smarinier.net>
+ *
+ * @license GNU AGPL version 3 or any later version
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+namespace OCA\Workspace\Service\Group;
+
+use OCP\IGroupManager;
+
+class ConnectedGroupsService {
+
+	private static $LINKED_SPACE_GROUPS = [
+		'SPACE-U-2' => ['Scientists', 'Chemists']
+	];
+
+	private static $LINKED_GROUPS_IN = [
+
+	];
+
+	public function __construct(private IGroupManager $groupManager) {
```

Comment:
Could you declare `$groupManager` as private attribute, please?

Reply:
???

Reply:
Oups, it's me, I misread :upside_down_face: